resolwe_bio/tests/test_processor.py

```diff
@@ -1,12 +1,15 @@
 # pylint: disable=missing-docstring
 from django.test import TestCase
 
-from server.models import Processor, iterate_schema
+try:
```

Comment:
Instead of ```get_processes``` we can have this try/catch in ```utils.py``` and then import the Process from ```utils.py``` directly here.